api-reference/coupons/coupon-object.mdx

```diff
@@ -1,106 +1,5 @@
 ---
-title: "The coupon object"
-description: "This object represents a coupon. This coupon can be applied to a customer with overriden values."
+title: " The coupon object"
```

Comment:
Why adding this space at the beginning everytime?

Reply:
Good catch - fixed the issues 🙌